linera-chain/src/manager.rs

```diff
@@ -455,6 +453,7 @@ impl ChainManager {
         validated: ValidatedBlockCertificate,
         key_pair: Option<&KeyPair>,
         local_time: Timestamp,
+        blobs: BTreeMap<BlobId, Blob>,
```

Comment:
I'm sure you've already answered this a couple of times but I keep forgetting:

Why do we have to pass the `blobs` here if they should be contained within the block which is part of the `validated` above?

Reply:
The block contains only hashes, not the full blobs.

Reply:
Only for _read_ blobs. For the published ones it should contain the blobs themselves.

Reply:
The `Block`/`ExecutedBlock` structs never contain the blobs, neither the published nor the read ones. The `BlockProposal` does contain the published ones.

Reply:
Ah, right. I know remember asking the same question 2 weeks ago.